src/constants/SelectCategory.jsx

```diff
@@ -0,0 +1,22 @@
+const SERVICES = [
```

Comment:
If you moved it to the `src/constants/SelectCategory.js` (the `.js` file if it is not ` .jsx` file - I mean a component, should be named with a lowercase) it should be still in any other file. Besides, you have this component also in `components`.